Resources/public/scss/modules/content-tree/_list.item.scss

```diff
@@ -4,7 +4,9 @@ $list-item-height: calculateRem(20px);
 #{$list-item} {
     line-height: $list-item-height;
     max-height: $list-item-height;
-
+    overflow: hidden;
```

Comment:
Why overflow hidden? I think scroll would be better.

Reply:
Scroll is on parent container .m-tree__scrollable-wrapper so we don't need another scroll